src/plugins/kibana_react/public/code_editor/languages/helpers.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { monaco } from '@kbn/monaco';
+import { LangModuleType } from './';
+
+function registerLanguage(language: LangModuleType) {
```

Comment:
the same question for `registerLanguage`. I think we should have one point for registering languages in the '@kbn/monaco' module.